bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx

```diff
@@ -1,7 +1,6 @@
 import React, { Component } from 'react';
 import injectWbResizeEvent from '/imports/ui/components/presentation/resize-wrapper/component';
 import YouTube from 'react-youtube';
-import Vimeo from 'react-vimeo';
```

Comment:
`    "react-vimeo": "^2.0.0",` would then be free to be removed from package.json, with package-lock.json consequences. @lfzawacki pls confirm this is no longer required

Reply:
Yes, you can remove it, this was never used.